Adding 'Updating Terraform Enterprise License' section to Application Administration section (#1777)
hashicorp/terraform-website: content/source/docs/cloud/run/api.html.md

### 2. Create the File for Upload

> The [configuration version API](../api/configuration-versions.html) requires a `tar.gz` file to be uploaded in order for the configuration version to be used for a run, so the content directory (the directory containing the Terraform configuration) must be packaged into a `tar.gz` file.

Reviewer: ```suggestion
The [configuration version API](../api/configuration-versions.html) requires an uploaded `tar.gz` file to use the configuration version for a run, so you must package the directory containing the Terraform configuration into a `tar.gz` file.
```
